data/sql/updates/pending_db_world/rev_1735350584050913500.sql

```diff
@@ -0,0 +1,7 @@
+--
+-- Removes NOT_REPEATABL` and WHILE_CHARMED event_flag, allowing to always drop the flag no matter if it's CC'd or not.
+UPDATE smart_scripts SET `event_flags` = `event_flags` &~1 &~512 WHERE `entryorguid` = 23576 AND `source_type` = 0 AND `id` = 1;
```

Comment:
Back ticks

Reply:
Done

Reply:
Also changed from 2 indiviual values to grouped

![image](https://github.com/user-attachments/assets/531545cc-1e5b-451b-b032-fc7d58901ea1)